frontend/src/Components/Page/Header/SeriesSearchResult.css

```diff
@@ -10,6 +11,18 @@
   object-fit: contain;
 }
 
+.posterSfw {
+  width: 80px;
```

Comment:
to avoid duplicating this css in several spots we should add a prop on SeriesPoster for `blur` or something, move this to SeriesPoster.css and then just set the prop when we need it.